Guide for using Ledger devices with Ignition (#5130)
NomicFoundation/hardhat: docs/src/content/ignition/docs/guides/ledger.md

> # Deploying with Ledger hardware wallet

Reviewer: I think I prefer `Deploying with a Ledger hardware wallet`. What do you think?

Hardhat Ignition supports deploying contracts using a Ledger hardware wallet via the `hardhat-ledger` plugin. This guide will show you how to deploy your contracts using a Ledger device.

> The first step will be to install the `hardhat-ledger` plugin:

Reviewer: ```suggestion
The first step is to install the `hardhat-ledger` plugin:
```
